packages/integration-karma/test/synthetic-shadow/keyframes/index.spec.js

```diff
@@ -0,0 +1,31 @@
+import { createElement } from 'lwc';
+import A from 'x/a';
+import B from 'x/b';
+
+describe('keyframes', () => {
+    it('animation applies to element with local keyframes', () => {
+        const el = createElement('x-a', { is: A });
+        document.body.appendChild(el);
+
+        const div = el.shadowRoot.querySelector('div');
+
+        // if the div is actually animating, then its opacity will be between 0 and 0.9
+        expect(parseFloat(getComputedStyle(div).opacity)).toBeLessThan(1);
+
+        if (process.env.NATIVE_SHADOW) {
+            expect(getComputedStyle(div).animationName).toEqual('fade');
+        } else {
+            expect(getComputedStyle(div).animationName).toEqual('fade-x-a_a');
+        }
```

Comment:
I don't think we should assert the animation name itself as the scoped name is an implementation detail. As long as the element is animated, the test should pass.

Reply:
Removed.